english/water-dreams/fish-sanctuary.md

```diff
@@ -0,0 +1,2 @@
+Ploop! What a rush! hey guys! you scream out too all the other waterdrops in the sacturary, 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.